src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -3372,7 +3373,7 @@ node.Type switch
                 RegexNode.Bol => "Match if at the beginning of a line.",
                 RegexNode.Boundary => $"Match if at a word boundary.",
                 RegexNode.Capture when node.N != -1 => $"{DescribeNonNegative(node.M)} capturing group. Uncaptures the {DescribeNonNegative(node.N)} capturing group.",
-                RegexNode.Capture when node.N == -1 => $"{DescribeNonNegative(node.M)} capturing group.",
+                RegexNode.Capture when node.N == -1 => $"{DescribeNonNegative(node.M)} capturing group" + (string.IsNullOrEmpty(capName) ? "." : $" (\"{capName}\")."),
```

Comment:
This should use the `Literal(...)` helper to output the `capName`.  That not only handles adding the quotes, it ensures characters in the string are rendered appropriately for C# source.

We also need to do a similar rendering for the case when node.N != -1, both for the capture group and the uncapture group.